### PR TITLE
Future dir: blog post. Clarify cert rotation.

### PR DESCRIPTION
**Pull request by garloff**
Signed-off-by: Kurt Garloff <kurt@garloff.de>

### CODE REVIEW

File: Release-Notes/Release3.md [hunk 1]
```diff
@@ -124,7 +124,11 @@ docker-compose is then no longer available and docker compose must be used inste
 
 ## Resolved Issues
 
-* certificate rotation in k8s clusters
+* Certificates in k8s clusters can expire (typically after one year).
```
Comment (fkr) [inline Release-Notes/Release3.md:127]: please without ( )

* Certificates in k8s clusters are subject to expiration - typically after one year.

File: Release-Notes/Release3.md [hunk 2]
```diff
@@ -172,6 +176,9 @@ updates for R3 at the end of April 2023.
 
 ## Future directions (selected Highlights)
```
Comment (fkr) [inline Release-Notes/Release3.md:177]: Am not really sure this belongs here. Where is the added value of this section? Once we publish the blog post it is there - before that no reason to talk about it.

**Review by fkr — CHANGES_REQUESTED** [review]
see inline comments

File: Release-Notes/Release3.md (hunk 2 above)
Comment (garloff) [reply Release-Notes/Release3.md:177]: OK, then remove this section completely.

**Comment by garloff** [comment]
Removed section on Future directions.

Added acknowledgement to community and upstream communities.

File: Release-Notes/Release3.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ tang from the state of a technical preview to be fully supported.
   (https://github.com/SovereignCloudStack/k8s-cluster-api-provider/)
   has seen major managability improvements.
   Please consult the
-  [k8s capi provider release notes](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/master/Release-Notes-R3.md)
+  [k8s cluster api provider release notes](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/master/Release-Notes-R3.md)
```
Comment (itrich) [inline Release-Notes/Release3.md:21]: We switched to `main` branch

**Review by itrich — CHANGES_REQUESTED** [review]

File: Release-Notes/Release3.md (hunk 3 above)
Comment (garloff) [reply Release-Notes/Release3.md]: Fixed, thanks for noticing!

**Review by itrich — APPROVED** [review]
LGTM now!

File: Release-Notes/Release3.md (hunk 1 above)
Comment (garloff) [reply Release-Notes/Release3.md]: OK, reworded.

**Pull request merged**